sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/Manager.java

```diff
@@ -25,8 +26,9 @@
     protected Manager(HttpPipeline httpPipeline, AzureProfile profile, InnerT innerManagementClient) {
         this.httpPipeline = httpPipeline;
         if (httpPipeline != null) {
-            this.resourceManager = ResourceManager.authenticate(httpPipeline, profile)
-                .withDefaultSubscription();
+            ResourceManager.Configurable resourceConfigurable = ResourceManager.configure();
```

Comment:
is there way to make this part of code resuable?

Reply:
The method `withInternalHttpPipeline` can be reused by each manager. But for `.authenticate`, it has to be called in each manager itself.